### PR TITLE
Add preview support for OSD vertical elements

### PR DESCRIPTION
**Pull request by fiam**
Also, add preview for OSD_VARIO

### CODE REVIEW

File: tabs/osd.js [hunk 1]
```diff
@@ -667,8 +672,18 @@ OSD.GUI.preview = {
         //noinspection JSUnresolvedVariable
         var field_id = parseInt(ev.dataTransfer.getData('text'));
         var display_item = OSD.data.display_items[field_id];
-        var overflows_line = FONT.constants.SIZES.LINE - ((position % FONT.constants.SIZES.LINE) + display_item.preview.length);
-
+        var line_width = 0;
+        var item_width = 0;
+        for (var ii = 0; ii < display_item.preview.length; ii++) {
+            if (display_item.preview[ii] == '\n'.charCodeAt(0)) {
+                item_width = Math.max(item_width, item_width);
```
Comment (DzikuVx) [inline tabs/osd.js]: This doesn't makes sense

File: tabs/osd.js [hunk 2]
```diff
@@ -667,8 +672,18 @@ OSD.GUI.preview = {
...
+                line_width = 0;
+                continue;
+            }
+            line_width++;
+        }
+        item_width = Math.max(item_width, item_width);
```
Comment (DzikuVx) [inline tabs/osd.js]: The same?

**Comment by fiam** [comment]
@DzikuVx Nice catch! I have a bunch of uncommitted stuff in my working copy, so I manually edited the patch and I ended up messing up that line, which then I copied onto the other one. I'll amend the commit right now.

**Comment by DzikuVx** [comment]
@fiam I think your amend did not amended anything `Math.max` is still there. Just keep it simple and make another commit :) `You should not lie to GIT` :)

**Comment by fiam** [comment]
@DzikuVx I stashed all my changes, applied this patch and made the changes over it to avoid making the same error (not that it would surprise me if I still made the same error 😄 ). `Math.max()` is there to set the element width to the maximum width calculated for each of its lines. Note that I changed the second argument from `item_width` to `line_width`, so the line actually does something now.

**Comment by DzikuVx** [comment]
I trust you know what you are doing :)

**Pull request merged**